packages/client/src/client.js

```diff
@@ -309,29 +309,28 @@ export default class PostRPCClient {
 				messages.push('response: ' + JSON.stringify(response))
 			}
 
-			if (response && response.id) {	// Call
+			// Payload is in response to a call
+			if (response && response.id) {
 				if (this._logging) {
 					messages.push('call response')
 				}
 
 				for (let i = this.queue.length - 1; i >= 0; i--) {
 					const call = this.queue[i]
 
-					// Match to queue
+					// Found matching call in the queue, begin processing
 					if (response.id === call.id) {
 						const result = response.hasOwnProperty('result') ? response.result : null
 						const error = response.hasOwnProperty('error') ? response.error : null
 
-						if (call.callback !== null) {
+						if (typeof call.callback === 'function') {
```

Comment:
when i pass a callback that's not a function, still get an uncaught error in the timeout handler
<img width="712" alt="Screen Shot 2019-09-03 at 4 54 34 PM" src="https://user-images.githubusercontent.com/2358736/64213880-9c437580-ce6b-11e9-95f4-b7ad78099f69.png">

Reply:
thanks. fixed.